backend-base/src/test/java/org/eclipse/che/incubator/workspace/telemetry/base/MockBaseConfiguration.java

```diff
@@ -0,0 +1,127 @@
+package org.eclipse.che.incubator.workspace.telemetry.base;
+
+import com.google.common.annotations.VisibleForTesting;
+import io.quarkus.test.Mock;
+import org.eclipse.che.api.core.*;
+import org.eclipse.che.api.core.rest.*;
+import org.eclipse.che.api.core.rest.shared.dto.Link;
+import org.eclipse.che.incubator.workspace.telemetry.MockHttpJsonResponse;
+
+import javax.enterprise.inject.Produces;
+import javax.validation.constraints.NotNull;
+import java.io.*;
+import java.util.List;
+import java.util.Map;
+
+@Mock
+public class MockBaseConfiguration extends BaseConfiguration {
+
+    private class MockHttpRequest implements HttpJsonRequest {
+
+        private String mockResponse;
+
+        public MockHttpRequest() {
+            mockResponse = this.getMockResponse();
+            if (mockResponse.equals("")) {
+                System.err.println("No mock response found");
```

Comment:
could be addressed in a separate PR, but later we should probably add a logger

Reply:
`org.slf4j` seems to be already available in the project btw

Reply:
You are right, this requires a bigger refactor.  We are instantiating loggers ourselves instead of injecting them, and when we inject them we need to move the constructor logic into `@PostConstruct`.  I'll open a PR for this.